routes/auth/auth.go

```diff
@@ -247,7 +247,7 @@ func Setup(m *macaron.Macaron) {
 			m.Get("/auth/github", RequiresIntegrationSetting, reqSignIn, GithubLogin)
 
 			m.Get("/user/list", reqSignIn, reqManager, ListUsers)
-			m.Get("/user/show/:id", reqSignIn, reqManager, Show)
+			m.Get("/user/show/:id", reqSignIn, reqSignIn, Show)
```

Comment:
mmm... I don't remember correctly but without this normal user doesn't retrieve his data.
I need to study again how macaron handle authentication, I don't remember correctly but I remember that I fixed this to fix retrieve of detail for a normal user.

Reply:
Probably, simply we could leave only reqSignIn because it is the first handler of the authentication pipeline. Before my changes, the second reqManager was blocking the normal user. :)